app/src/main/java/com/lucasfagundes/android_pokedex/view/MainActivity.kt

```diff
@@ -2,23 +2,29 @@ package com.lucasfagundes.android_pokedex.view
 
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
-import androidx.recyclerview.widget.LinearLayoutManager
-import com.lucasfagundes.android_pokedex.R
 import com.lucasfagundes.android_pokedex.databinding.ActivityMainBinding
 import com.lucasfagundes.android_pokedex.domain.Pokemon
+import com.lucasfagundes.android_pokedex.domain.PokemonType
 
-private var _binding: ActivityMainBinding? = null
-private val binding: ActivityMainBinding get() = _binding!!
+private lateinit var binding: ActivityMainBinding
 
 class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_main)
+        binding = ActivityMainBinding.inflate(layoutInflater)
+        setContentView(binding.root)
 
         val recyclerView = binding.pokemonRecyclerView
 
-        val layoutManager = LinearLayoutManager(this)
-        recyclerView.layoutManager = layoutManager
-
+        val charmander = Pokemon(
+            "https://assets.pokemon.com/assets/cms2/img/pokedex/detail/214.png",
+            4,
+            "Charmander",
+            listOf(PokemonType("Fire"))
+        )
+        val pokemons = listOf(
```

Comment:
nao é padrão de programação nomear um paramentro de lista apenas colocando o plural no nome dos objetos que ela contem. O padrão seria
```suggestion
        val pokemonList = listOf(
```